### DISCUSSION TITLE
最近的更新日志找不到了？

### DISCUSSION THREAD

**Discussion by justincnn** [linweiyuan/go-chatgpt-api#182] [General]
原来的更新日志会在[README.md](https://github.com/linweiyuan/go-chatgpt-api/discussions/new?category=general#readme) 里面有说明什么时候更新的内容，

最近改版之后，都不知道更新了什么了？还是希望大神可以放进去，谢谢啊

**Comment by linweiyuan** [comment]
不加 https://github.com/linweiyuan/go-chatgpt-api/discussions/117

**Comment by linweiyuan** [comment]
每次提交代码自动打包新镜像，如果一天多次提交，更新日志写到吐

再说，过期的更新日志也没什么参考价值啊

除非重大不兼容旧版的改动，就置顶几天，日志看的人会看，不看的人也不会看
